modules/userId/index.js

```diff
@@ -711,6 +716,20 @@ function initSubmodules(submodules, consentData) {
   }, []);
 }
 
+function updateInitializedSubmodules(submodule) {
+  let updated = false;
+  for (let i = 0; i < initializedSubmodules.length; i++) {
+    if (submodule.config.name.toLowerCase() === initializedSubmodules[i].config.name.toLowerCase()) {
+      updated = true;
+      initializedSubmodules[i] = submodule;
+    }
```

Comment:
Can we break out of this for loop if there's a match?

Reply:
updated